enterprise/views.py

```diff
@@ -1249,7 +1251,8 @@ def get(self, request, enterprise_uuid, course_id):
             # In case of Audit course modes enroll the learner directly through
             # enrollment API client and redirect the learner to dashboard.
             enrollment_api_client.enroll_user_in_course(
-                request.user.username, course_id, selected_course_mode['slug']
+                request.user.username, course_id, selected_course_mode['slug'], cohort=None,
+                enterprise_uuid=str(enterprise_customer_user.enterprise_customer.uuid)
```

Comment:
[style nitpick] No need to include `cohort=None` here.

Reply:
@iloveagent57  It's done